airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/ConfigPersistenceBuilderTest.java

```diff
@@ -155,20 +155,6 @@ public void testCreateDbPersistenceWithoutSetupDatabase() throws Exception {
         dbPersistence.dumpConfigs());
   }
 
-  @Test
-  public void testCreateFileSystemConfigPersistence() throws Exception {
```

Comment:
why delete this test?

Reply:
The `ConfigPersistenceBuilder#getFileSystemPersistence` method is redundant. So this method is deleted, together with this test.

Reply:
makes sense!